src/api/rate_limiter.py

```diff
@@ -22,12 +22,22 @@ def __init__(self, request, log=None, message="", msg_args=()):
             message = f"Request %s: %s + %s {message}"
             log(message, request.method, request.url, request.cookies)
 
+class RLRequestFailed(RateLimiterError):
+    def __init__(self, request, *args, **kwargs):
+        super().__init__(request, *args, **kwargs)
+
+class RLRequestClientError(RLRequestFailed):
+    def __init__(self, request, *args, **kwargs):
+        super().__init__(request, *args, **kwargs)
```

Comment:
Maybe `RLClientError`, `RLClientError4xx` would be a better name?
(same with Server)

Reply:
also the name `RLRequestServerError` wrongly give the impression of inheriting from `RLRequestFailed`